src/client/features/interactions/index.js

```diff
@@ -0,0 +1,135 @@
+const React = require('react');
+const h = require('react-hyperscript');
+const _ = require('lodash');
+const queryString = require('query-string');
+const Loader = require('react-loader');
+
+const make_cytoscape = require('../../common/cy/');
+
+const { ServerAPI } = require('../../services/');
+
+const { BaseNetworkView } = require('../../common/components');
+const { getLayoutConfig } = require('../../common/cy/layout');
+
+class Interactions extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cy: make_cytoscape({ headless: true }),
```

Comment:
you should:
* create a new stylesheet for interactions specifically and inject it into this make cytoscape function
* modify the make cytoscape function to look for a ```stylesheet``` arg and send that arg to the cytoscape constructor if found. Default to the sbgn stylesheet if not found